geode-benchmarks/src/main/java/org/apache/geode/benchmark/tests/ReplicatedIndexedQueryBenchmark.java

```diff
@@ -50,7 +50,7 @@ public void run() throws Exception {
 
   @Override
   public TestConfig configure() {
-    TestConfig config = GeodeBenchmark.createConfig();
+    TestConfig config = TestConfigFactory.build();
     config.threads(Runtime.getRuntime().availableProcessors() * 2);
```

Comment:
Several tests have this line which will override the user defined thread count with default.